model-optimizer/mo/utils/cli_parser.py

```diff
@@ -845,6 +845,23 @@ def get_freeze_placeholder_values(argv_input: str, argv_freeze_placeholder_with_
     return placeholder_values, input_node_names
 
 
+def parse_dimension(dim: str):
+    if '..' in dim:
+        numbers_reg = r'[0-9]+'
+        dims = dim.split('..')
+        min_val = np.int64(dims[0]) if re.match(numbers_reg, dims[0]) else np.int64(0)
```

Comment:
I think that we should complain on a wrong format if `re.match` returns None. please add it

Reply:
of course, except empty dims[0]

Reply:
According to syntax description in the ticket min or max bound cannot be specified as -1 or ?. If some bound is omitted we should use "..10" or "5.." record, for example "[5,..6,2..,?]". So I added raise if re.match returns None, but didn't allow ? or -1 as a boundary.